CHANGELOG.md

```diff
@@ -47,6 +47,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Changed `automatic casting` for LoggerConnector `metrics` ([#5218](https://github.com/PyTorchLightning/pytorch-lightning/pull/5218))
 
 
+- Changed the default of `find_unused_parameters` to `False` in DDP ([#5185](https://github.com/PyTorchLightning/pytorch-lightning/pull/5185))
+
+
```

Comment:
i wonder if you'd like to pull this out to another PR that's super easy to review and merge (just changing the defaults)

Reply:
sounds good! I reverted the changes and moved them to #5435